external/sparc64/README.md

```diff
@@ -1,60 +0,0 @@
-NuPIC Core external libraries
```

Comment:
We now automatically build all externals from sources; these instructions are now unnecessary and don't reflect the current state of the build.